omero/users/cli/chgrp.txt

```diff
@@ -0,0 +1,140 @@
+-----------------------------
+
+The :omerocmd:`chgrp` command moves objects between groups. Further help is
+available using the :option:`-h` option::
+
+    $ bin/omero chgrp -h
+
+The :command:`chgrp` command will remove entire graphs of objects based on the
+IDs of the topmost objects. The command can be modified to include the movement
+of objects that would, by default, be excluded or exclude objects that would,
+by default, be included using the :option:`--include` and :option:`--exclude`
+options.
+
+Unlike the :ref:`delete command <cli_delete>`, objects of the three annotation
+types, `FileAnnotation`, `TagAnnotation` and `TermAnnotation` are not moved
+by default when the objects to which they are linked are moved.
+
+It is also possible to move objects lower in the hierarchy by specifying
+the type and ID of a topmost object and the type of the lower object. For
+instance, moving all of the images under a given project.
+
+By default the command confirms the movement of the target objects but
+it can also provide a detailed report of all the moved objects via a
+:option:`--report` option. A :option:`--dry-run` option can be used to report
+on what objects would be moved without actually moving them.
+
+Examples
+^^^^^^^^
+
+Basic move
+==========
+
+::
+
+    $ bin/omero chgrp 5 OriginalFile:101
+    $ bin/omero chgrp Group:5 Project:51
+    $ bin/omero chgrp ExperimenterGroup:5 Project:51
+    $ bin/omero chgrp lab_group Project:51
+
+In the first line, the original file with ID 101 will be moved to the group
+with ID 5. In the second, project 51 will be moved to group 5 including any
+datasets inside only that project and any images that are only contained within
+any moved datasets. If group 5 is named 'lab_group' then the last two lines
+will have the same effect as the second.
+
+Moving multiple objects
+=======================
+
+Multiple objects can be specified with each type being followed by an ID
+or a comma-separated list of IDs. The order of objects or IDs is not
+significant, thus all three calls below are identical in moving
+project 51 and datasets 53 and 54 to group 5.::
+
+    $ bin/omero chgrp 5 Project:51 Dataset:53,54
+    $ bin/omero chgrp 5 Dataset:54,53 Project:51
+    $ bin/omero chgrp 5 Dataset:53 Project:51 Dataset:54
+
+Moving lower level objects
+==========================
+
+To move objects below a specified top-level object the following form
+of the object specifier is used.::
+
+    $ bin/omero chgrp 5 Project/Dataset/Image:51
+
+Here the all of images under the project 51 would be moved. It is
+necessary to specify intermediate objects in the hierarchy and so::
+
+    $ bin/omero chgrp 5 Project/Image:51
+
+would have the same effect as the call above. Links can also be moved
+and so::
+
+$ bin/omero chgrp 5 Project/DatasetImageLink:51 Dataset/DatasetImageLink:53
+
+would effectively orphan all images under project 51 and dataset 53 that are
+not also under other datasets.
```

Comment:
Yes, it is something you told me specifically about! I'm happy to not mention it though.